concordia/templates/transcriptions/campaign_topic_list.html

```diff
@@ -0,0 +1,44 @@
+{% extends "base.html" %}
+
+{% load staticfiles %}
+
+{% block title %}Campaigns{% endblock title %}
+
+{% block head_content %}
+    <link rel="canonical" href="https://{{ request.get_host }}{{ request.path }}">
+{% endblock head_content %}
+
+{% block breadcrumbs %}
+    <li class="breadcrumb-item active" aria-current="page">Campaigns</li>
+{% endblock breadcrumbs %}
+
+{% block main_content %}
+<div class="container py-3">
+    <h1 class="sr-only">Campaigns</h1>
+    <ul class="list-unstyled">
+        {% for campaign in campaigns_topics %}
+            <li class="p-4 mb-1 bg-footer">
+                <h2 class="h1 mb-3">{{ campaign.title }}</h2>
+                <div class="row">
+                    <a class="col-md-5 order-md-2"
+                    {% if campaign.model_type == "Campaign" %}
```

Comment:
Could we replace this with a `get_absolute_url()` call? That would appear to remove the need to set the `model_type` property in both cases.

Reply:
Yes, I'll do that right now.